tehuti.py

```diff
@@ -168,6 +170,45 @@ def run(self):
         return [sum(vals) / self.number for vals in self.usage_log]
 
 
+class AccuracyMetric(Metric):
+    class Context(object):
+        pass
+
+    def __init__(self, body, setup=None, repeat=100, number=1, name=None):
```

Comment:
> Less code = less bugs.
> 
> Less code, fewer bugs :wink:

If only I'd done that deliberately, to demonstrate "fewer words = fewer bugs". :unamused: